doc/api/colors_api.rst

```diff
@@ -11,39 +11,157 @@ The Color :ref:`tutorials <tutorials-colors>` and :ref:`examples
    :no-members:
    :no-inherited-members:
 
-Classes
--------
+Color Conversion tools
+----------------------
+
+.. autosummary::
+   :toctree: _as_gen/
+   :template: autosummary.rst
+
+   is_color_like
+   same_color
+
+   hsv_to_rgb
+   rgb_to_hsv
+
+   to_hex
+   to_rgb
+   to_rgba
+   to_rgba_array
+
+   get_named_colors_mapping
+
+
+.. _norms_and_colormaps:
+
+Normalization and Colormapping
+------------------------------
+
+Some `~.artists.Artist` classes can map an array of input data to RGBA
+values, (ex `~.axes.Axes.scatter` or `~.axes.Axes.imshow`).  The
+machinery for this is implemented via the `~.cm.ScalarMappable` base
+class in `~.cm` and the `~.Normalize` and `~.Colormap` classes in
+`~.colors` (this module).
+
+At the core, colormapping is going from a scalar value to a RGB tuple
+(formally :math:`f(x) : ℝ^1 \rightarrow ℝ^3`).  To effectively
+communicate through the color we want pick a :ref:`colormap suited to
+the data <sphx_glr_tutorials_colors_colormaps.py>` but in general
+"good" colormaps smoothly and continuously change their RGB values as
+a function of the input data.  By looking at the RGB values as we go
+through the full range of the user data (e.g. a colorbar) we can trace
```

Comment:
Also the "good" in parenthesis is probably a stand in for some precise term and otherwise not true for categorical data and questionable on discrete data.